common/transport/amqp/package.json

```diff
@@ -8,7 +8,7 @@
   "typings": "index.d.ts",
   "dependencies": {
     "async": "^2.6.2",
-    "azure-iot-common": "1.10.1",
+    "azure-iot-common": "^1.10.1",
```

Comment:
>"^ [](start = 24, length = 2)

please keep the strict version declaration for intra-sdk package dependencies

Reply:
I removed all the carets for our packages.